SE-XXX Typed throws.md

```diff
@@ -438,10 +467,16 @@ do {
     dump(error)
     recoverFromLibraryError(error)
 }
-
 ```
 
-Scenarios where we can make use of the Swift compiler with the current proposal:
+The most evident scenario is that no longer is needed to catch a certain type in the `catch` clauses, due to the fact that the generic `catch` clause will become strongly typed with the type specified in the function signature.

```

Comment:
The most evident scenario for a do block with a single thrown error inside is that it is no longer needed to catch a certain type in the `catch` clauses, due to the fact that the generic `catch` clause will become strongly typed with the error type specified in the throwing function's signature.